src/lib/Components/Bidding/Screens/SelectMaxBid.tsx

```diff
@@ -113,8 +113,8 @@ export const MaxBidScreen = createRefetchContainer(
     `,
   },
   graphql`
-    query SelectMaxBidRefetchQuery($saleArtworkID: String!) {
-      sale_artwork: saleArtwork(id: $saleArtworkID) {
+    query SelectMaxBidRefetchQuery($saleArtworkNodeID: ID!) {
+      node(id: $saleArtworkNodeID) {
         ...SelectMaxBid_sale_artwork
```

Comment:
@yuki24 @dleve123 Another FYI to triple-check in next QA. This is refetching the sale artwork node using the `node` root field with its global object ID.